### PR TITLE
cleaning up repo

### PR DESCRIPTION
**Pull request by DT-DawidWroblewski**
#### What type of PR is this?

Add one of the following kinds:

* cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

File: code/API_definitions/number_verification.yaml [hunk 1]
```diff
@@ -6,6 +6,8 @@ info:
     It also makes it possible for a Service provider to verify the number itself by returning the phone number associated to the authenticated user's access token.
     
     In this API **phone number** term refers to the mobile phone number
+    
+    [GSMA Mobile Connect Verified MSISDN specification](https://www.gsma.com/identity/wp-content/uploads/2022/12/IDY.54-Mobile-Connect-Verified-MSISDN-Definition-and-Technical-Requirements-1.0.pdf) was used as source of input for this API.  For more about Mobile Connect, please see [Mobile Connect website](https://mobileconnect.io/).
```
Comment (bigludo7) [inline code/API_definitions/number_verification.yaml:10]: I'm no confortable to keep MC link in the swagger. I will prefer to keep simple. For the mention of MC in the read.me is perfect.

**Review by bigludo7 — COMMENTED** [review]
Thanks !
Just one comment; I will prefer to not mention and link MC in the swagger. Else I'm fine.

File: code/API_definitions/number_verification.yaml (hunk 1 above)
Comment (bigludo7) [reply code/API_definitions/number_verification.yaml:10]: Please discard my comment - we have this sentence in SimSwap.yaml so for sake of consistency leet keep this for now.

**Review by bigludo7 — APPROVED** [review]
Let's go with this massive clean up and then we will check in the detail.

**Review by diegogonmar — APPROVED** [review]
Thanks @DT-DawidWroblewski. LGTM

**Review by hdamker — APPROVED** [review]

**Pull request merged**